be/src/storage/tablet_meta_manager.cpp

```diff
@@ -1788,7 +1792,7 @@ Status TabletMetaManager::pending_rowset_iterate(DataDir* store, TTabletId table
     put_fixed64_le(&prefix, BigEndian::FromHost64(tablet_id));
 
     return store->get_meta()->iterate(
-            META_COLUMN_FAMILY_INDEX, prefix, [&](std::string_view key, std::string_view value) -> bool {
+            META_COLUMN_FAMILY_INDEX, prefix, [&](std::string_view key, std::string_view value) -> StatusOr<bool> {
```

Comment:
why not return Status directly. It is a little weird to return StatusOr<bool>.
Status::OK can be considered as True.

Reply:
It's different, because return `true` or `false` means if we need to break this iterate or not. And return Status::OK or ERROR means we have issue when ierate.

And, this seems weird, it should return ERROR instead of false:
```
if (!decode_meta_pending_rowset_key(key, &tid, &version)) {
                    LOG(ERROR) << "corrupt pending rowset key: " << hexdump(key.data(), key.length());
                    return false;
                }
```

Reply:
I will fix this.

Reply:
There are three possibilities here:
1. the iteration breaks normally because the data we want was found. Return Status Ok with false
2. An unintended error occurs during the execution of the iteration. Returns Status Error 
3. The iteration proceeds normally. Returns Status Ok with true